csrc/velox/functions/gpt2_bpe_tokenizer.cpp

```diff
@@ -0,0 +1,405 @@
+#include "gpt2_bpe_tokenizer.h"
```

Comment:
nit: maybe put into `functions/text`?  Simliar to https://github.com/facebookresearch/torcharrow/pull/242/files#diff-17c8133a1b9bc8be7b62d7983c0264777d894e46b0bf6add7c7c10ace70b3d5cR1

Reply:
Done!